base/base.jl

```diff
@@ -278,3 +281,5 @@ immutable Nullable{T}
     Nullable(value::T) = new(false, value)
 end
 
+get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
```

Comment:
I guess these functions are needed to include string.jl and gmp.jl? Can't you simply move `include("nullable.jl")` above them in sysimg.jl?